bundles/org.openhab.binding.miio/README.md

```diff
@@ -3265,6 +3266,18 @@ e.g. `openhab:send actionCommand 'upd_timer["1498595904821", "on"]'` would enabl
 | rgbColor             | Color                | RGB Color                                |            |
 | name                 | String               | Name                                     |            |
 
+### Mi Smart LED Bulb Essential (White and Color) (<a name="yeelink-light-color5">yeelink.light.color5</a>) Channels
+
+| Channel              | Type                 | Description                              | Comment    |
+|----------------------|----------------------|------------------------------------------|------------|
+| power                | Switch               | Power                                    |            |
+| brightness           | Dimmer               | Brightness                               |            |
+| delayoff             | Number:Time          | Shutdown Timer                           |            |
+| colorTemperature     | Number               | Color Temperature                        |            |
+| colorMode            | Number               | Color Mode                               |            |
```

Comment:
Can you give a short description how this can be used. E.g. the value range.

Reply:
Maybe the requestor @migue74 can test and confirm. I don't own such light.
My understanding is that this is for selecting RGB, white or other modes

Reply:
I realised the options were already in the file... just not highlighted in the readme.